### PR TITLE
Add performance baseline for m6a.metal

### PR DESCRIPTION
**Pull request by zulinx86**
## Changes

Adds missing baselines for long-running performance tests for m6a.metal.

## Reason

Currently, the following baselines for long-running performance tests are missing for m6a.metal
- vsock_throughput
- network_tcp_throughput
- block_performance
- network_latency

## License Acceptance

By submitting this pull request, I confirm that my contribution is made under
the terms of the Apache 2.0 license.

## PR Checklist

- [x] All commits in this PR are signed (`git commit -s`).
- [x] If a specific issue led to this PR, this PR closes the issue.
- [x] The description of changes is clear and encompassing.
- [x] Any required documentation changes (code and docs) are included in this PR.
- [x] New `unsafe` code is documented.
- [x] API changes follow the [Runbook for Firecracker API changes][2].
- [x] User-facing changes are mentioned in `CHANGELOG.md`.
- [x] All added/changed functionality is tested.

---

- [ ] This functionality can be added in [`rust-vmm`][1].

[1]: https://github.com/rust-vmm
[2]: https://github.com/firecracker-microvm/firecracker/blob/main/docs/api-change-runbook.md

### CODE REVIEW

**Comment by zulinx86** [comment]
Should this PR add a description to CHANGELOG same as https://github.com/firecracker-microvm/firecracker/pull/3224#pullrequestreview-1160982173 ?

**Comment by dianpopa** [comment]
> Should this PR add a description to CHANGELOG same as [#3224 (review)](https://github.com/firecracker-microvm/firecracker/pull/3224#pullrequestreview-1160982173) ?

Yes, let s add a CHNAGELOG entry.

File: tests/integration_tests/performance/configs/test_snap_restore_performance_config_4.14.json [hunk 1]
```diff
@@ -2182,350 +2182,350 @@
...
                                                 }
                                             }
                                         },
                                         "5vcpu_128mb": {
                                             "P50": {
                                                 "restore": {
-                                                    "target": 7.605,
-                                                    "delta_percentage": 15
+                                                    "target": 14.283,
+                                                    "delta_percentage": 17
```
Comment (luminitavoicu) [inline tests/integration_tests/performance/configs/test_snap_restore_performance_config_4.14.json:2242]: This is quite a big increase. Let's decouple adding missing baselines from the updates and investigate this latency bump.

Comment (zulinx86) [reply tests/integration_tests/performance/configs/test_snap_restore_performance_config_4.14.json]: That's right. As this PR aims to add missing performance baselines for AMD (m6a.metal), let's decouple! Thanks!

**Review by pb8o — APPROVED** [review]

**Review by luminitavoicu — APPROVED** [review]

**Pull request merged**